mm2src/mm2_main/tests/docker_tests/docker_tests_common.rs

```diff
@@ -1063,11 +1099,37 @@ pub fn slp_supplied_node() -> MarketMakerIt {
     .unwrap()
 }
 
+fn exec_in_container(container_id: &str, command: &str) -> String {
+    let output = Command::new("docker")
+        .args(["exec", container_id, "sh", "-c", command])
+        .output()
+        .expect("failed to execute command in container");
+
+    String::from_utf8_lossy(&output.stdout).trim().to_string()
+}
+
+fn find_container_id_by_port(port: u16) -> Option<String> {
+    let output = Command::new("docker")
+        .args(["ps", "--filter", &format!("publish={}", port), "--format", "{{.ID}}"])
+        .output()
+        .expect("failed to execute docker ps");
+
+    let container_id = str::from_utf8(&output.stdout).ok()?.trim().to_string();
+    if container_id.is_empty() {
+        None
+    } else {
+        Some(container_id)
+    }
+}
```

Comment:
Could you clarify why is this needed? Once the container is started we have `DockerNode` type that contains the container informations.

Reply:
It technically is not. We could pass the DockerNode variable here.